software/signature-pdf.yml

```
@@ -0,0 +1,14 @@
+name: "Signature PDF"
+website_url: "https://pdf.24eme.fr/"
```

To match other projects with a demo as the main website
```suggestion
website_url: "https://github.com/24eme/signaturepdf"
```